web/alembic/versions/20251112_1320-3fe755434ebe_add_jobsets_and_evaluations.py

```diff
@@ -0,0 +1,73 @@
+"""Add jobsets and evaluations
+
+Revision ID: 3fe755434ebe
+Revises: ebd80c41f648
+Create Date: 2025-11-12 13:20:04.853554+00:00
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '3fe755434ebe'
+down_revision: Union[str, Sequence[str], None] = 'ebd80c41f648'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
(might be nice to add a `CONTRIBUTING.md` to point to the Alembic docs on how to do this ;) )